build.xml

```diff
@@ -29,6 +29,12 @@
         </exec>
     </target>
 
+    <target name="create-index" depends="prepare" description="Create index">
```

Comment:
tak prave slo o to aby si sa s tym nemusel babrat rucne ... idealne by bolo keby "create index" volala kazda komponenta zavisla na syrupe ...  potom by sa ti nestalo, ze nemozes pustit testy, lebo mas spatny index.